Dockerfile

```
@@ -122,6 +122,9 @@ ENV PATH="${PATH}:/home/renovate/python3.10/bin"
 RUN $PYENV_ROOT/plugins/python-build/bin/python-build $(pyenv latest -f -k 3.13) $HOME/python3.13
 ENV PATH="${PATH}:/home/renovate/python3.13/bin"
 
+# Install jsonnet-bundler
+RUN go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest
```

```suggestion
RUN go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest && go clean -cache -modcache
```
This will remove the caches which is safe after the binary installation.

---

Thank you :+1: